db/schema.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 11.6
--- Dumped by pg_dump version 11.6
+-- Dumped from database version 11.2
+-- Dumped by pg_dump version 11.10
```

Comment:
Would be cool to leave these versions intact because I think we're running 11.6 on our deploy

Reply:
Updated.